src/assets/js/fetcher.js

```diff
@@ -52,14 +52,14 @@ class Fetcher {
         },
         body: JSON.stringify(data),
       });
-      return res.status;
-    } catch (e) {
-      console.log('error: ', e);
-      if (this.failOnError && this.isError(e)) {
-        console.log(`got error → failing over`);
-        // reload();
+      const responseObject = await res.json();
```

Comment:
This will display generic network errors. This easily happens when deleting the pmo-access cookie while on the /a/create page and then trying to submit.

I filed a ticket and we'll solve it later.